sparse_strips/vello_hybrid/examples/render_to_file.rs

```diff
@@ -157,28 +157,26 @@ async fn run() {
     device.poll(wgpu::Maintain::Wait);
 
     // Read back the pixel data
-    let mut img_data = Vec::with_capacity(usize::from(width) * usize::from(height) * 4);
-    for row in texture_copy_buffer
+    let mut pixmap = Pixmap::new(width, height);
+    for (row, buf) in texture_copy_buffer
         .slice(..)
         .get_mapped_range()
         .chunks_exact(bytes_per_row as usize)
+        .zip(pixmap.data_mut().chunks_exact_mut(usize::from(width) * 4))
     {
-        img_data.extend_from_slice(&row[0..width as usize * 4]);
+        buf.copy_from_slice(&row[0..usize::from(width) * 4]);
```

Comment:
This now starts zeroed and writes over, right? Rather than before where it started uninitialised, and writes once.

Given the context of this example, the difference is probably fine?

Reply:
That's right. For a one-shot case like this that may be a bit of a performance hit. With a constructor as in https://github.com/linebender/vello/pull/964/files#r2075398135 we could do the same as before but have it checked.

Reply:
Changed back with such a constructor, https://github.com/linebender/vello/pull/964/commits/f7bfd3787442413452ca396ffc01ae3e8d285f31.